specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```
@@ -0,0 +1,771 @@
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden\n* InvalidRequest - Maximum allowed length across all profile enrollments is 300 seconds.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "examples": {
+              "application/json": {
+                "code": "InvalidRequest",
+                "message": "Maximum allowed length across all profile enrollments is 300 seconds."
+              }
+            }
+          },
+          "404": {
+            "$ref": "#/responses/NotFound"
+          },
+          "409": {
+            "description": "Conflict\n* Conflict - Sending concurrent requests on same profile is not allowed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "examples": {
+              "application/json": {
+                "code": "Conflict",
+                "message": "Sending concurrent requests on same profile is not allowed."
+              }
+            }
+          },
+          "415": {
+            "description": "UnsupportedMediaType  - Unsupported media type. Only \"audio/wav; codecs=audio/pcm\" is accepted.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "examples": {
+              "application/json": {
+                "code": "UnsupportedMediaType",
+                "message": "Unsupported media type. Only 'audio/wav; codecs=audio/pcm' is accepted."
+              }
+            }
+          },
+          "429": {
+            "$ref": "#/responses/RateLimit"
+          },
+          "500": {
+            "$ref": "#/responses/ServerError"
+          }
+        }
+      }
+    },
+    "/text-independent/profiles/identifySingleSpeaker": {
+      "post": {
+        "description": "Identifies who is speaking in input audio among a list of candidate profiles.\n\nLimitations:\n* Maximum audio length is **120 seconds**\n* Minimum audio speech length is **20 seconds**\n* Maximum candidate speakers count is **50**\n* Minimum candidate speakers count is **1**\n* Minimum audio Signal-to-noise ratio (SNR) is **2dB**",
+        "summary": "Identify Profile",
+        "tags": [
+          "text-independent"
+        ],
+        "consumes": [
+          "audio/wav; codecs=audio/pcm"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "profileIds",
+            "type": "string",
+            "required": true,
+            "description": "Comma-delimited profile IDs. Maximum supported number is 50 IDs."
+          },
+          {
+            "in": "query",
+            "name": "ignoreMinLength",
+            "type": "boolean",
+            "description": "If true, the minimum amount of speech needed for identification is skipped. Default is false."
```

Default can/should be made part of the formal definition:
```suggestion
            "description": "If true, the minimum amount of speech needed for identification is skipped.",
            ”default”: false
```

---

done